From 83bcc5974633b1784768d9ab88603e33d35437d7 Mon Sep 17 00:00:00 2001
From: mohit <mohit.p@variscite.com>
Date: Thu, 14 Jan 2021 14:48:09 -0800
Subject: [PATCH 1/2] camera: It reverts below limitation.

fix max resolution to 720p:
bead54bb3aee96aa926436857e03a5593db26db8
---
 libcamera3/ImxCamera.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/libcamera3/ImxCamera.cpp b/libcamera3/ImxCamera.cpp
index d0d699c..8169c4d 100644
--- a/libcamera3/ImxCamera.cpp
+++ b/libcamera3/ImxCamera.cpp
@@ -96,9 +96,7 @@ status_t ImxCamera::initSensorStaticData()
     int previewCnt = 0, pictureCnt = 0;
     struct v4l2_frmsizeenum cam_frmsize;
     struct v4l2_frmivalenum vid_frmval;
-    char value[PROPERTY_VALUE_MAX] = {0};
 
-    property_get("ro.build.product", value, "0");
 
     while (ret == 0) {
         memset(TmpStr, 0, 20);
@@ -143,9 +141,6 @@ status_t ImxCamera::initSensorStaticData()
             mPreviewResolutions[previewCnt++] = cam_frmsize.discrete.width;
             mPreviewResolutions[previewCnt++] = cam_frmsize.discrete.height;
         }
-	if(cam_frmsize.discrete.height == 720 &&
-           (strncasecmp("som_mx8mn", value, strlen(value)) == 0))
-            break;
     }  // end while
 
     mPreviewResolutionCount = previewCnt;
-- 
2.25.1

